example/css/example.scss

```diff
@@ -141,3 +141,20 @@ input {
     margin-top: 0;
   }
 }
+.singleDateRange{
```

Comment:
Mind providing a screenshot? :+1: 

Could we place spaces before the `{` - we don't have a css linter currently! :(